src/generated/resources/assets/versatileportals/lang/zh_cn.json

```diff
@@ -0,0 +1,19 @@
+{
+  "block.versatileportals.portal": "传送门",
+  "block.versatileportals.portal.message.noPortalController": "找不到此门的传送门控制器",
+  "block.versatileportals.portal_controller": "传送门控制器",
+  "block.versatileportals.portal_controller.gui.color.b": "传送门色彩 B：",
+  "block.versatileportals.portal_controller.gui.color.g": "传送门色彩 G：",
+  "block.versatileportals.portal_controller.gui.color.r": "传送门色彩 R：",
+  "block.versatileportals.portal_controller.gui.slotLabel.controlItem": "维度标识器放置位置",
+  "block.versatileportals.portal_controller.message.invalidControlItem.hasNoKey": "没有维度标识器，传送门无法工作",
```

Comment:
This translation key no longer exist, and a few others have been changed. Please double check that you have all the correct translation keys. Use https://github.com/mrp-v2/VersatilePortals/blob/master/src/generated/resources/assets/versatileportals/lang/en_us.json as a reference.